ci/linux-direct-centos8-docker-gcc-release.jenkinsfile

```diff
@@ -44,9 +44,9 @@ node(node_label) {
                 load '.ci/lib/config-release.jenkinsfile'
 
                 load '../ci/stage-build-nosgx.jenkinsfile'
-                load '../ci/stage-test.jenkinsfile'
                 load '../ci/stage-test-direct.jenkinsfile'
                 load '../ci/stage-test-sandstone.jenkinsfile'
+                load '../ci/stage-test.jenkinsfile'
```

Comment:
what is this change for? generally unrelated refactoring should be done in a separate PR.

Reply:
stage-test.jenkinsfile has workload_parser in the end which parses all the workload results in the end. Since CI examples has separate file, that's why this code was needed at the end.